src/routers/alpha-router/functions/best-swap-route.ts

```diff
@@ -467,7 +467,6 @@ export async function getBestSwapRouteBy(
   // For each gas estimate, normalize decimals to that of the chosen usd token.
   const estimatedGasUsedUSDs = _(bestSwap)
     .map((routeWithValidQuote) => {
-      // TODO: will error if gasToken has decimals greater than usdToken
```

Comment:
We discussed this is a change of behavior, and would like to get feedback on the original author(s) (cc @snreynolds @zhongeric ) whether we'd like to pursue this path. The comment says erroring it out in such case, and we'd want to understand better the reasoning behind it.